src/wp-includes/global-styles-and-settings.php

```diff
@@ -369,5 +402,6 @@ function wp_clean_theme_json_cache() {
 	wp_cache_delete( 'wp_get_global_stylesheet', 'theme_json' );
 	wp_cache_delete( 'wp_get_global_settings_custom', 'theme_json' );
 	wp_cache_delete( 'wp_get_global_settings_theme', 'theme_json' );
+	wp_cache_delete( 'wp_get_global_custom_css', 'theme_json' );
```

Comment:
```suggestion
	wp_cache_delete( 'wp_get_global_styles_custom_css', 'theme_json' );
```

Reply:
For consistency, use the function's name that is setting the cache. In this case, `wp_get_global_styles_custom_css(0` is setting the cache.